core/templates/vector.h

```diff
@@ -157,40 +158,40 @@ class Vector {
 		return ret;
 	}
 
-	Vector<T> slice(int p_begin, int p_end = INT_MAX) const {
+	Vector<T> slice(Size p_begin, Size p_end = INT64_MAX) const {
```

Comment:
Need a macro for the max as well, I'd suggest a `SIZE_MAX/MIN` and `USIZE_MAX/MIN` though might clash with definitions existing already